app/models/manageiq/providers/vmware/infra_manager/template.rb

```diff
@@ -4,4 +4,8 @@ class ManageIQ::Providers::Vmware::InfraManager::Template < ManageIQ::Providers:
   def cloneable?
     true
   end
+
+  def validate_provisioning
+    ExtManagementSystem.where(:id => ems_id).exists?
```

Comment:
why was the check for `ems_id` being `nil` removed before we make the SQL call?
/cc @kbrock